internal/backends/winit/winitwindowadapter.rs

```diff
@@ -122,10 +122,13 @@ pub struct WinitWindowAdapter {
     window_level: Cell<winit::window::WindowLevel>,
 
     pub(crate) renderer: Box<dyn WinitCompatibleRenderer>,
-    // We cache the size because winit_window.inner_size() can return different value between calls (eg, on X11)
-    // And we wan see the newer value before the Resized event was received, leading to inconsistencies
+    /// We cache the size because winit_window.inner_size() can return different value between calls (eg, on X11)
+    /// And we wan see the newer value before the Resized event was received, leading to inconsistencies
     size: Cell<PhysicalSize>,
 
+    /// Whether the size has been set explicitly
+    has_explicit_size: Cell<bool>,
```

Comment:
Maybe this should be called `has_fixed_size`? Or the comment should say _who_ has set the size. For example "/// Whether the size has been set explicitly via set_size."